app/services/statements/query.rb

```diff
@@ -1,13 +1,21 @@
 module Statements
   class Query
-    def initialize(lead_provider:)
+    def initialize(lead_provider:, cohort_start_years: nil, updated_since: nil)
       @lead_provider = lead_provider
+      @cohort_start_years = cohort_start_years&.split(",")
+      @updated_since = updated_since
     end
 
     def statements
-      Statement
-        .includes(:cohort)
-        .where(lead_provider:)
+      scope = Statement
+                .includes(:cohort)
```

Comment:
Any reason we're using `#includes` here over `#eager_load` - is there any reason we wouldn't want  PostgreSQL to do a left join?

Reply:
I think leaving the decision to `includes` depending on what's best is probably my preference here for performance rather than being explicit with `eager_load`